tools/show_segment_status/README

```diff
@@ -0,0 +1,4 @@
+操作步骤:
+1. conf中修改相应的集群配置，可以指定database, table, be三个维度进行查看
+2. 运行python show_segment_status.py去获取相应的结果
```

Comment:
English, dude :)

Reply:
And explain why we need this script? When should we use it?  What is the output ？

Reply:
Done.